### ISSUE TITLE
Call to undefined function password_hash() thrown setting user/pass.

### ISSUE THREAD

**Issue by mpmc** [quidsup/notrack#83]
System:
Raspberry Pi (B+ 512M).
Raspbian (installed using [raspbian-ua-netinst](https://github.com/debian-pi/raspbian-ua-netinst))

Issue: Attempting to set user & pass results in the above exception on PHP versions < 5.5.0. The script installed  5.4.45-0+deb7u4 from the Raspbian repo.

```
2016-08-18 11:56:26: (log.c.166) server started
2016-08-18 11:56:39: (server.c.1558) server stopped by UID = 0 PID = 3749
2016-08-18 11:56:40: (log.c.166) server started
2016-08-18 11:56:58: (server.c.1558) server stopped by UID = 0 PID = 3945
2016-08-18 11:57:00: (log.c.166) server started
2016-08-18 12:01:16: (mod_fastcgi.c.2683) FastCGI-stderr: PHP Fatal error:  Call to undefined function password_hash() in /opt/notrack/admin/config.php on line 1061

2016-08-18 12:01:31: (mod_fastcgi.c.2683) FastCGI-stderr: PHP Fatal error:  Call to undefined function password_hash() in /opt/notrack/admin/config.php on line 1061

2016-08-18 12:01:34: (mod_fastcgi.c.2683) FastCGI-stderr: PHP Fatal error:  Call to undefined function password_hash() in /opt/notrack/admin/config.php on line 1061

2016-08-18 12:01:42: (mod_fastcgi.c.2683) FastCGI-stderr: PHP Notice:  Undefined offset: 2 in /opt/notrack/admin/config.php on line 281
PHP Notice:  Undefined offset: 2 in /opt/notrack/admin/config.php on line 281
PHP Notice:  Undefined offset: 2 in /opt/notrack/admin/config.php on line 281

2016-08-18 12:02:19: (mod_fastcgi.c.2683) FastCGI-stderr: PHP Fatal error:  Call to undefined function password_hash() in /opt/notrack/admin/config.php on line 1061

```

**Comment by quidsup** [comment]
This will be fixed in the release of v0.8

**Comment by quidsup** [comment]
Issue fixed
